pyproject.toml

```diff
@@ -123,10 +123,6 @@ version-file = "fmriprep/_version.py"
 # Developer tool configurations
 #
 
-# Disable black
-[tool.black]
-exclude = ".*"
-
```

Comment:
Please leave this. It's specifically to prevent IDEs that auto-apply black from modifying files at save.